### PR TITLE
4.6 Release Notes for Network Edge

### PR DESCRIPTION
**Pull request by bmcelvee**
Adding release notes for Network Edge
4.6 Release Notes Tracker: https://github.com/openshift/openshift-docs/issues/22406

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -754,6 +754,48 @@ The tests must perform an environment configuration every time they are executed
...
+
+[id="ocp-4-6-ingress-nlb-aws"]
+==== Ingress Controller Network Load Balancer for AWS
+
+Configuration of an Ingress Controller Network Load Balancer (NLB) for new and existing AWS clusters is now supported.
+
+See xref:../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws-network-load-balancer.adoc#configuring-ingress-cluster-traffic-aws-network-load-balancer[Configuring ingress cluster traffic on AWS using a Network Load Balancer] for more information.
+
+[id="ocp-4-6-ingress-route53"]
+==== Ingress Operator endpoint configure for AWS Route53
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:793]: Perhaps `configuration` in this context would be better than `configure`?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -2055,6 +2097,8 @@ In the table below, features are marked with the following statuses:
 
 * The Network Interface Card (NIC) validation phase fails because it does not wait for the devices on the node to become available. The waiting time for a pod to start running on the node is too short, so a pod could still have the status `Pending` and be incorrectly tested. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890088[*BZ#1890088*])
 
+* The `ose-egress-dns-proxy` image has a known defect that prevents the container from starting. This image is also known to have been broken in earlier releases, so this is not considered a regression in 4.6. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888024[*BZ#1888024*])
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2100]: s/is also known to have been broken/is broken

**Review by codyhoag — DISMISSED** [review]
Two nits; otherwise LGTM!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (bmcelvee) [reply release_notes/ocp-4-6-release-notes.adoc:793]: Oops. That's what I actually meant to type. Thank you!

**Review by codyhoag — APPROVED** [review]

**Pull request merged**